src/services/edusharing/services/EduSharingConnectorV7.js

```diff
@@ -221,11 +227,51 @@ class EduSharingConnector {
 		}
 	}
 
+	async getPlayerForNode(nodeUuid) {
+		const url = `${ES_ENDPOINTS.RENDERER}${nodeUuid}`;
+		const options = {
+			method: 'GET',
+			url,
+			headers: {
+				Accept: 'application/json',
+			},
+		};
+		const user = Configuration.get('ES_USER');
+
+		const response = await this.eduSharingRequest(options, user);
+		const parsed = JSON.parse(response);
+		if (parsed && parsed.detailsSnippet && typeof parsed.detailsSnippet === 'string') {
```

Comment:
if (parsed && typeof parsed.detailsSnippet === 'string') {  
Should do the same.